src/main/java/com/redhat/labs/omp/resource/VersionResource.java

```diff
@@ -21,21 +24,45 @@
  *
  */
 @RequestScoped
-@Path("/api/v1/version")
+@Path("/api")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
 public class VersionResource {
     @Inject
     VersionService versionService;
     
     @GET
-    @Timed(name="versionResourceTimer")
-    @Counted(name="versionResourceCounter")
+    @Path("/v1/version")
+    @Timed(name="versionResourceV1Timer")
+    @Counted(name="versionResourceV1Counter")
     @PermitAll
-    public VersionManifest getVersion() {
+    @Deprecated
+    @Operation(deprecated = true, summary = "Returns version 1 of the manifest.")
+    public VersionManifest getVersionV1() {
         VersionManifest vm = versionService.getVersionManifest();
         vm.addContainer(versionService.getGitApiVersion());
 
         return vm;
     }
+
+    @GET
+    @Path("/v2/version")
```

Comment:
I think I kind of messed up on this one. In the config service I used a header for versioning https://github.com/rht-labs/lodestar-backend/blob/master/src/main/java/com/redhat/labs/omp/resource/ConfigResource.java#L40 but in this class I use the url. How much effort would it be to line up the usage to use the header pattern?

Reply:
I don't think it would be hard to switch it to the header.  I think we need to keep the `/api/v1/version` because that is currently what the FE is using.  I'll make the switch, keep the existing as deprecated, and then we can remove it once the FE makes their update.